NBXplorer/Extensions.cs

```diff
@@ -156,6 +157,17 @@ public static AuthenticationBuilder AddNBXplorerAuthentication(this Authenticati
 			});
 		}
 
+		public static IServiceCollection AddMessageBrokers(this IServiceCollection services,IConfiguration config )
+		{
+			//There must be a better way to do this ! IoC container should be able to inject config.
+
+			if (!string.IsNullOrWhiteSpace(config["asbcnstr"]))
+				services.AddSingleton<IHostedService, AzureServiceBus>();
```

Comment:
Please add this inside AddNBXplorer, Always add this hosted service. Just make sure that it does nothing when it starts if no connection strings have been set. (As you can resolve the configuration object in the ctor of the class)